cassandane/Cassandane/Cyrus/JMAPMailbox.pm

```diff
@@ -1375,6 +1375,79 @@ sub test_mailbox_set_order
     $self->assert_null($res->[0][1]{notDestroyed});
 }
 
+sub test_mailbox_move_to_deleted_parent
+    :min_version_3_8 :needs_component_jmap
```

Comment:
I think 3.6 will want a backport of this too, so can you make the test `:min_version_3_6` please?

```suggestion
    :min_version_3_6 :needs_component_jmap
```

Reply:
I've done this.  Thanks ;)